package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
none of these files (package, node_modules etc) should be in the main directory, only in the scenario_editor directory. I suspect its because you tried to run npm i in the main dir. Should be deleted